app.js

```diff
@@ -107,7 +107,9 @@ app.get('/components/:component/:example*?/preview', function (req, res, next) {
     {{ ${macroName}(${macroParameters}) }}`
   )
 
-  res.render('component-preview')
+  res.render('component-preview', {
+    bodyClasses: req.query.iframe && 'app-iframe-in-component-preview'
```

Comment:
Is this conditionally adding the `app-iframe-in-component-preview` only if `req.query.iframe` is truthy? I've not seen this construct before and it's a bit hard to understand what's going on.

If that's what it's doing, would definitely prefer using ternary instead, i.e.

```js
bodyClasses: req.query.iframe ? 'app-iframe-in-component-preview' : '';
```

Reply:
Ah fair enough, I'm used to this but definitely more boring ways to do this, will update.